covid_model/model.py

```diff
@@ -12,22 +12,23 @@ class CovidModel(ODEBuilder, CovidModelSpecifications):
                         'age': ['0-19', '20-39', '40-64', '65+'],
                         'vacc': ['none', 'shot1', 'shot2', 'shot3'],
                         'priorinf': ['none', 'non-omicron', 'omicron'],
-                        'variant': ['none', 'omicron'],
-                        'immun': ['none', 'imm0', 'imm1', 'imm2', 'imm3']})
+                        'variant': ['none', 'alpha', 'delta', 'omicron'],
+                        # 'immun': ['none', 'imm0', 'imm1', 'imm2', 'imm3']})
+                        'immun': ['none', 'weak', 'strong']})
 
     param_attr_names = ('age', 'vacc', 'priorinf', 'variant', 'immun')
 
     default_end_date = dt.date(2022, 5, 31)
 
-    def __init__(self, base_model=None, **spec_args):
+    def __init__(self, base_model=None, deepcopy_params=True, **spec_args):
```

Comment:
In some situations, it was causing problems to copy the model without deepcopying the params. But...it turns out that to deepcopy the params on the full model, even with some optimization, takes like 100 seconds. They're huge. So I have an option not to do it, which is used in the incremental model fitting.